tests/services/order_validation_service/test_case_sample_rules.py

```diff
@@ -370,3 +372,27 @@ def test_missing_volume_no_container(valid_order: OrderWithCases):
 
     # THEN no error should be returned
     assert not errors
+
+
+def test_validate_sex_subject_id_clash(valid_order: OrderWithCases, sample_store: Store):
```

Comment:
Have we changed the type of the valid_order? A bit uncertain of what the order type is but I like seeing which order type it is valid for.

Reply:
As the previous function was for Balsamic it had BalsamicOrder, should it be something else?

Reply:
I would type hint it as specifically as possible - so in this case it would still be a BalsamicOrder (although we have several fixtures called valid_order so we should make sure it is still the same fixture). I just know that there are multiple different `valid_order` fixtures which look different depending on what order type there is so type hinting it as OrderWithCases does not give me enough to go by here. We might want to rename them to valid_balsamic_order, valid_tomte_order etc.

Reply:
resolöved in person, leave a better type hinting for another PR